src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/_help.py

```diff
@@ -12,7 +12,22 @@
     short-summary: Access to a secure, dedicated cache for your Azure applications
 """
 
+helps['redis export'] = """
+    type: command
+    short-summary: Export data stored in a redis cache. 
+"""
+
+helps['redis import-method'] = """
+    type: command
+    short-summary: Import data into a redis cache.
```

Comment:
@johanste is that actually what this command does? If so, why is it not simple called `az redis import`?